translations/ai_rules.tex/de.po

```diff
@@ -88,6 +103,15 @@ msgid ""
 "Any differences to the above will be described in any given Scenario's own rules.\n"
 "\n"
 msgstr ""
+"Bot-Helden müssen verdeckte Gebiete wie üblich entdecken, indem sie 1 BP ausgeben, bevor sie sich auf sie bewegen.\n"
+"Der Spieler wählt die Ausrichtung des Gebiets.\n"
+"Bot-Helden können nicht kapitulieren und du kannst vor ihnen nicht kapitulieren;\n"
+"sie werden immer kämpfen, bis sie keine Einheiten mehr haben.\n"
+"Wenn man einen Kampf gegen einen Bot-Helden gewinnt, erhält man keine Belohnungen, außer dies ist im Szenario angegeben.\n"
+"Bot-Helden haben weder ein Stadt- oder Heldentableau noch Ressourcen.\n"
+"Ihre Einheiten sind statisch und werden durch den Aufbau des Szenarios oder durch andere Regeln definiert.\\par\n"
+"Alle Abweichungen von den oben genannten Regeln werden in den Regeln des jeweiligen Szenarios beschrieben.\n"
+"\n"
 
 #. type: multicols*
 #: sections/ai_rules.tex:32 sections/ai_rules.tex:115 sections/combat.tex:185
```

Comment:
It's the one just below here.

po format is designed to dedup strings, so that the same original string is translated to the same string. But with the way how we use it with po4a, it makes some of these strings which are actually formatting, hard to "translate" without affecting other parts of the book